messaging-modules/spring-apache-camel/pom.xml

```diff
@@ -77,12 +77,17 @@
             <version>${camel.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-spring-xml</artifactId>
+            <version>${camel.version}</version>
+        </dependency>
     </dependencies>
 
     <properties>

```

Comment:
Since now 3.12.0 is used, let's delete extra env.camel.version

Reply:
env.spring.version is used here:
![image](https://github.com/eugenp/tutorials/assets/12120641/05b81040-3e5c-4c85-97b3-1dc4b11049de)

Reply:
Ok, could we have <version>${spring.version}</version> in line 43?
And also adjust the <properties>

Also replace env.camel.version with camel.version (also adjust <properties>)

Reply:
Just to rename that property variable ${spring.version}?
${spring.version} is not accessible from parent dependency. It is a spring-boot dependency.